src/borg/helpers.py

```diff
@@ -1494,6 +1494,17 @@ def file_status(mode):
     return '?'
 
 
+def chunk(it, size):
+    """
+    Chunk an iterator <it> into pieces of <size>.
+
+    >>> list(chunker('ABCDEFG', 3))
+    [['A', 'B', 'C'], ['D', 'E', 'F'], ['G']]
+    """
+    iterable = iter(it)
+    return iter(lambda: list(islice(iterable, size)), [])
```

Comment:
now that our cython-based iterators work like standard iterators, this actually works. \o/